### PR TITLE
feat: add to evm module

### PR DESCRIPTION
**Pull request by nowooj**

### CODE REVIEW

File: content/en/docs/develop/core-modules/evm.md [hunk 1]
```diff
@@ -0,0 +1,935 @@
+---
+title: EVM
+weight: 170
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/evm.md]: the documents are alphabetically ordered. `weight` should be `90` and other module pages are also needed to be updated accordingly.

Comment (nowooj) [reply content/en/docs/develop/core-modules/evm.md]: https://github.com/xpladev/docs/pull/14/commits/35fe8b84d38b932eda47b07fa551eeaba4a10195

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**